cfme/containers/image.py

```diff
@@ -133,23 +163,24 @@ def step(self):
     def resetter(self):
         # Reset view and selection
         self.view.toolbar.view_selector.select("List View")
-        self.view.paginator.reset_selection()
+        self.view.paginator.set_items_per_page(1000)
 
 
 @navigator.register(Image, 'Details')
 class Details(CFMENavigateStep):
     VIEW = ImageDetailsView
-    prerequisite = NavigateToSibling('All')
+    prerequisite = NavigateToAttribute("parent", 'All')
 
     def step(self):
+
         self.prerequisite_view.entities.get_entity(provider=self.obj.provider.name,
                                                    id=self.obj.id).click()
 
 
 @navigator.register(Image, 'EditTags')
 class ImageRegistryEditTags(CFMENavigateStep):
     VIEW = TagPageView
-    prerequisite = NavigateToSibling('Details')
+    prerequisite = NavigateToAttribute("parent", 'Details')
```

Comment:
**Required**

This navigation call doesn't look right to me - 'Details' is registered against the `Image` class, as is 'EditTags', making them sibling destinations.  Shouldn't be a need for a change to this prerequisite

Reply:
I am sorry , Can you please elaborate exactly what are you suggest me to change?
I changed this prerequisite according to Pete suggestion that way it use in collection implementation!

Reply:
@brahmanim: EditTags is on the same object as 'Details'. So the navigate step should be prerequeisite = NavigateToSibiling('Details'). This is what @mshriver  means by sibling destinations. What you have here is equivalent to navigate_to(self.parent,'Details').

Reply:
Thank you! I change it back to prerequisite = NavigateToSibling('Details')